### DISCUSSION TITLE
Ability to port newly created pandapower networks to pypower and PyPSA

### DISCUSSION THREAD

**Discussion by AD9881327** [e2nIEE/pandapower#1391] [Q&A] [answered]
I have created several networks in pandapower API and I would also like to run them in pypower and pypsa, i have seen the pandapower converters which can be used to port the networks to pypower and pypsa. It is possible to port newly created networks or only the ones saved in the networks folders of pandapower? Can somebody please help me troubleshoot this or provide a probable solution.

**Comment by ascheidl** [comment, marked as answer]
you can export any newly created network to pypower using the function pandapower.converter.to_ppc.

However, I can not imagine any reason why one would want to use pypower or pypsa instead of pandapower ;)

**Comment by AD9881327** [comment]
can you please provide a short example, I can follow the same procedure as this is a huge roadblock for me,
I have tried the function but it always says that such a grid does not exist in "networks".
I want to compare the power flows and learn the converter functionality.

it would be of immense help to me,

**Comment by ascheidl** [comment]
an example can be found in the docstring:

[link](https://github.com/e2nIEE/pandapower/blob/6545808f49e8b35ae9eb694930db171a3977540d/pandapower/converter/pypower/to_ppc.py#L75)

you can use the function with any pandapower network
